src/components/integrated/OakPrimaryNav/OakPrimaryNav.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+import { OakFlex } from "@/components/base/OakFlex";
+import { OakLI } from "@/components/base";
+import {
+  OakPrimaryNavItem,
+  OakPrimaryNavItemProps,
+} from "@/components/integrated/OakPrimaryNavItem";
+
+export type OakPrimaryNavProps = {
+  ariaLabel?: string;
+  navItems: OakPrimaryNavItemProps[];
+};
+
+export const OakPrimaryNav = ({ ariaLabel, navItems }: OakPrimaryNavProps) => {
+  return (
+    <nav aria-label={ariaLabel}>
+      <OakFlex as={"ul"}>
+        {navItems.map(
+          ({ href, isCurrent, label, ...rest }: OakPrimaryNavItemProps) => (
+            <OakLI
+              $listStyle={"none"}
+              $mr={["space-between-m", "space-between-m", "space-between-none"]}
```

Comment:
this doesn't seem to make sense . We end up with larger margins on the side and the bottom for the mobile view whilst none for desktop ? Also we end up with some unnecessary margin on the rhs after the final element this way. Could we instead use the $gap property of parent flex and also the flex direction ?

Reply:
updated